src/parser/mod.rs

```diff
@@ -7198,6 +7429,15 @@ impl<'a> Parser<'a> {
         }
     }
 
+    pub fn parse_optional_string(&mut self) -> Result<Option<String>, ParserError> {
+        if self.consume_token(&Token::Comma) {
+            let str = self.parse_literal_string()?;
+            Ok(Some(str))
+        } else {
+            Ok(None)
+        }
+    }
```

Comment:
for where this is used, we can add a function `parse_datetime_64()` that inlines this function instead?
otherwise the function is a bit misleading by requiring a Comma unrelated to advertised string in its signature.

Reply:
The function is updated.